pulseguardian/web.py

```diff
@@ -63,53 +63,12 @@
 # Role for admin user
 ADMIN_ROLE = 'admin'
 
-
-# Monkey-patch werkzeug.
-
-def generate_adhoc_ssl_pair(cn=None):
-    """Generate a 1024-bit self-signed SSL pair.
-    This is a verbatim copy of werkzeug.serving.generate_adhoc_ssl_pair
-    from commit 91ec97963c77188cc75ba19b66e1ba0929376a34 except the key
-    length has been increased from 768 bits to 1024 bits, since recent
-    versions of Firefox and other browsers have increased key-length
-    requirements.
-    """
-    from random import random
-    from OpenSSL import crypto
-
-    # pretty damn sure that this is not actually accepted by anyone
-    if cn is None:
-        cn = '*'
-
-    cert = crypto.X509()
-    cert.set_serial_number(int(random() * sys.maxint))
-    cert.gmtime_adj_notBefore(0)
-    cert.gmtime_adj_notAfter(60 * 60 * 24 * 365)
-
-    subject = cert.get_subject()
-    subject.CN = cn
-    subject.O = 'Dummy Certificate'
-
-    issuer = cert.get_issuer()
-    issuer.CN = 'Untrusted Authority'
-    issuer.O = 'Self-Signed'
-
-    pkey = crypto.PKey()
-    pkey.generate_key(crypto.TYPE_RSA, 1024)
```

Comment:
Confirmed -- pulseguardian is using `Werkzeug==0.14.1`, which has now raised to 2048:
https://github.com/pallets/werkzeug/blob/0.14.1/werkzeug/serving.py#L413